Analysis/source/bbtauTauAnalyzer.cxx

```diff
@@ -57,7 +38,7 @@ class bbtautauAnalyzer : public BaseEventAnalyzer {
         }
 
         for(auto wp_2 = working_points.rbegin(); wp_2 != working_points.rend(); ++wp_2) {
-            if(tau_2->Passed(TauIdDiscriminator::byIsolationMVArun2017v2DBoldDMwLT2017, *wp_2)) {
+            if(tau_2->Passed(signalObjectSelector.GetTauVSjetDiscriminator().first, *wp_2)) {
```

Comment:
L50-51: use `signalObjectSelector.GetTauVSjetDiscriminator().second` instead of `Medium`